### ISSUE TITLE
`$(opam var share)/merlin/vim/doc/tags` should be deleted on uninstall

### ISSUE THREAD

**Issue by ddickstein** [ocaml/merlin#1822]
The `:helptags` command in Vim that Merlin advises to run after installation generates a `tags` file in the `doc` directory. It should be cleaned up if Merlin is uninstalled.

**Comment by voodoos** [comment]
Good catch, I guess there is some field in the opam package to do that ?
